### PR TITLE
Add Salvator-XS-H3 r8a7795 with 8GiB (4 x 2 GiB) machine

### PR DESCRIPTION
**Pull request by dsemenets**
Signed-off-by: Dmytro Semenets <dmytro_semenets@epam.com>

### CODE REVIEW

**Comment by rshym** [comment]
@otyshchenko1 Could you please take a look?

File: templates/machine/salvator-xs-h3-4x2g-xt/50_bblayers.conf.inc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+BBLAYERS =+ "\
+  ${METADIR}/bsp/meta-renesas-rcar-gen3/meta-rcar-gen3 \
+  ${METADIR}/meta-xt-images-rcar-gen3 \
+  ${METADIR}/meta-xt-agl-base \
+  ${METADIR}/meta-aos \
```
Comment (otyshchenko1) [inline templates/machine/salvator-xs-h3-4x2g-xt/50_bblayers.conf.inc]: Curious question: Do we really need meta-aos here?

**Comment by otyshchenko1** [comment]
LGTM (with open question regarding the inclusion of meta-aos)

File: templates/machine/salvator-xs-h3-4x2g-xt/50_bblayers.conf.inc (hunk 1 above)
Comment (dsemenets) [reply templates/machine/salvator-xs-h3-4x2g-xt/50_bblayers.conf.inc]: Removed line

**Comment by rshym** [comment]
Tested by build of `domd-agl-cluster-demo-platform` of prod-cockpit-demo-src for `salvator-xs-h3-4x2g`.
Build passed OK.
meta-aos is not needed here, because it is provided from other recipe.

UPDATE: Tested by build together with https://github.com/xen-troops/meta-xt-prod-cockpit-demo/pull/3

`Acked-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>`

**Pull request merged**